ecosystem/typescript/sdk/src/aptos_types/account_address.ts

```diff
@@ -47,6 +47,29 @@ export class AccountAddress {
     return new AccountAddress(res);
   }
 
+  /**
+   * Checks if the string is a valid AccountAddress
+   * @param addr Hex string can be with a prefix or without a prefix,
+   *   e.g. '0x1aa' or '1aa'. Hex string will be left padded with 0s if too short.
+   */
+  static isValid(addr: MaybeHexString): boolean {
```

Comment:
An empty string would pass in the current implementation, which is wrong.

Reply:
Hmm... this is actually kinda interesting... isn't "" a compressed from of 0x0000...? I disagree :P

Reply:
"" is not a compressed form of 0x000, the most compressed form would be 0x0", so I'm fixing this.